### PR TITLE
Mon: Added network mode display to statusbar and hyperlink to ecal.ini

### PR DESCRIPTION
**Pull request by FlorianReimold**
Added hyperlink to `ecal.ini`:

grafik

Added colorful info in the status bar:

grafik

grafik

grafik

This PR is part of #563 and #556 

----------------------

⚠️ This PR already contains #568 ⚠️

### CODE REVIEW

**Comment by rex-schilasky** [comment]
@FlorianReimold may it sense to do the review now or do you want to remove the log and monitoring frequency states beforehand ?

**Comment by FlorianReimold** [comment]
Yes, I have converted the PR to a draft. Steps to complete:

- [ ] Remove bulky fequency stuff that nobody cares about from the status bar
- [ ] Finalize #568 (This PR needs the API from there)
- [ ] Merge the PR above into this branch
- [ ] ???
- [ ] Profit ✔️

**Comment by FlorianReimold** [comment]
This PR is ready for review, now

**Review by rex-schilasky — APPROVED** [review]

File: app/mon/mon_gui/src/widgets/system_information_widget/system_information_widget.cpp [hunk 1]
```diff
@@ -21,20 +21,207 @@
...
+  }
+
+  lines.push_front("<html><head/><body><pre>");
+  lines.push_back("</pre></body>");
+
+  return lines.join("\n");
 }
 
-SystemInformationWidget::~SystemInformationWidget()
+void SystemInformationWidget::openEcalIni(const QUrl& url)
```
Comment (Kerstin-Keller) [inline app/mon/mon_gui/src/widgets/system_information_widget/system_information_widget.cpp:114]: To be honest, not such a big fan of this function. It's massive. This Linux fallback, what is the drawback of always using it always?

What's the benefit of the current implementation that  `QDesktopServices::openUrl(url);` cannot do?

**Review by Kerstin-Keller — COMMENTED** [review]
In general I approve. However I'd like to understand what is the advantage of the complex logic in `openEcalIni` function.

**Review by Kerstin-Keller — APPROVED** [review]
😄

**Review by hannemn — APPROVED** [review]

**Pull request merged**